src/types/form.ts

```diff
@@ -41,6 +42,20 @@ export enum ResponseMode {
   Email = 'email',
 }
 
+export const Rating = _.range(1, 11)
```

Comment:
since the original array was an array of strings, should we convert these to strings?